app/resources/v1/pacbio/runs/library_resource.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module V1
+  module Pacbio
+    module Runs
+      class LibraryResource < JSONAPI::Resource
+        model_name 'Pacbio::Library'
+        attributes :state, :volume, :concentration, :template_prep_kit_box_barcode,
+                   :fragment_size, :created_at, :deactivated_at, :source_identifier
+      end
+    end
+  end
+end
```

Comment:
Why do we need a different library resource under the run namespace? Can't we just continue to use `app/resources/v1/pacbio/library_resource.rb`